### ISSUE TITLE
Comparison of tumour-based (Petersen Index) and inflammation-based (Glasgow Prognostic Score) scoring systems in patients undergoing curative resection for colon cancer

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#24]
## TL;DR

This paper compares the Petersen Index (PI) and the modified Glasgow Prognostic Score (mGPS) in patients undergoing curative resection of colon cancer. 

### Paper Link

https://www.nature.com/articles/6604926#Abs1

### Author/Institution

Roxburgh, C (University of Glasgow)

## Overview

Results of this study validate the use of the PI in predicting cancer-specific survival in patients undergoing elective potentially curative for Dukes’ B colon cancer. Furthermore, the results indicate that the mGPS further stratifies those patients with Dukes’ B and single node positive Dukes’ C cancers, classified as low risk by the PI. The PI and mGPS scoring systems could, therefore, be combined at a multi-disciplinary meeting to identify those patients with colorectal cancer who have undergone potentially curative surgery but are at high risk of cancer related death.

## Methods

colorectal cancer. survival analysis

## Cite

Roxburgh, C., Crozier, J., Maxwell, F. et al. Comparison of tumour-based (Petersen Index) and inflammation-based (Glasgow Prognostic Score) scoring systems in patients undergoing curative resection for colon cancer. Br J Cancer 100, 701–706 (2009). https://doi.org/10.1038/sj.bjc.6604926
